modules/cluster-logging-updating-logging.adoc

```diff
@@ -120,46 +97,155 @@ oc exec -n openshift-logging -c elasticsearch elasticsearch-cdm-1pbrl44l-1-55b75
+green  open   infra-000002                                                          Hz6WvINtTvKcQzw-ewmbYg   3 1       228994            0        296            148
+green  open   project.qitang.2c05acbd-bc12-4275-91ab-84d180b53505.2020.06.30        MUm3eFJjSPKQOJWoHskKqw   3 1        12262            0         14              7
+green  open   infra-000013                                                          KR9mMFUpQl-jraYtanyIGw   3 1       228166            0        298            148
+green  open   audit-000001                                                          eERqLdLmQOiQDFES1LBATQ   3 1            0            0          0              0
 ----
 
-.. Ensure that the Curator CronJob is using a 4.5 image:
+.. Verify that the log collector is updated to 4.5.:
 +
 ----
-$ oc get CronJob curator -n openshift-logging -o yaml |grep 'image:'
+$ oc get ds fluentd -o json | grep fluentd-init
 ----
 +
+You should see a `fluentd-init` container:
++
+----
+"containerName": "fluentd-init"
+----
+
+.. Verify that the log visualizer is updated to 4.5 using the Kibana CRD:
++
 ----
-image: registry.redhat.io/openshift4/ose-logging-curator5@sha256:330c3499e790d0e184414125a4843cd48849c601eb9f19ff82f30794c858b0bc
+$ oc get kibana kibana -o json
 ----
++
+There is a Kibana pod with the `ready` status:
++
+----
+[
+{
+"clusterCondition": {
+"kibana-5fdd766ffd-nb2jj": [
+{
+"lastTransitionTime": "2020-06-30T14:11:07Z",
+"reason": "ContainerCreating",
+"status": "True",
+"type": ""
+},
+{
+"lastTransitionTime": "2020-06-30T14:11:07Z",
+"reason": "ContainerCreating",
+"status": "True",
+"type": ""
+}
+]
+},
+"deployment": "kibana",
+"pods": {
+"failed": [],
+"notReady": [
+"kibana-5fdd766ffd-nb2jj"
+],
+"ready": []
+},
+"replicaSets": [
+"kibana-5fdd766ffd"
+],
+"replicas": 1
+}
+]
+----
+
+..Verify the curation is updated to 4.5. (There are elasticsearch-delete and elasticsearch-rollover index ):
++
+----
+$ oc get cronjob -o name
+----
++
+----
+cronjob.batch/curator
+cronjob.batch/elasticsearch-delete-app
+cronjob.batch/elasticsearch-delete-audit
+cronjob.batch/elasticsearch-delete-infra
+cronjob.batch/elasticsearch-rollover-app
+cronjob.batch/elasticsearch-rollover-audit
+cronjob.batch/elasticsearch-rollover-infra
+----
+
+.Post update
+
+After Cluster Logging and Elasticsearch Operators are fully updated to 4.5, you must perform the following actions to complete the upgrade.
+
+* Recreate your Kibana index patterns. Because of changes in the security plug-in, the cluster logging upgrade does not automatically create index patterns. 
+
+** Users must manually create index patterns to see logs for their projects. Because of the new Elasticsearch data model, all logs that were stored in an index prefixed with *project-* in {product-title} 4.4 are now co-located in a single index prefixed with *app-*. Users should create a new index patterns named `app` and use the `@timestamp` time field.
```

Comment:
How about using the content below as post action?

Because of the new Elasticsearch data model,  Users have to create index patterns to view the system and container logs. 

** The common users need to create index patterns using app* and `@timestamp` time field to view their container logs.
To view history data gathered by cluster logging 4.4,  in 4.4, Users need to create index patterns using 'project*' filed.

** Admin users need to create index patterns using the *app*, *infra*, and *audit* prefix with the `@timestamp` time field. 
To view history data gathered by  cluster logging 4.4, Admin users need to create index patterns using '.operation*' filed to view system logs and create index patterns using 'project*' field to view project-level logs.